wbooks/src/Books.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import myData from './books.json';
+import defaultBookIcon from './default_book.svg'
+import './Books.css';
+
+class Books extends Component {
+   constructor(props) {
+     super(props);
+     this.state = {
+       items: myData,
+     };
+     this.sortByField = this.sortByField.bind(this);
```

Comment:
This component won't need state or the sortByField function after the refactor suggested above.